pkg/reconciler/space/reconciler.go

```diff
@@ -366,7 +366,6 @@ func (r *Reconciler) reconcileLimitRange(desired, actual *v1.LimitRange) (*v1.Li
 func (r *Reconciler) reconcileServiceAccount(desired, actual *corev1.ServiceAccount) (*corev1.ServiceAccount, error) {
 	// Check for differences, if none we don't need to reconcile.
 	semanticEqual := equality.Semantic.DeepEqual(desired.ObjectMeta.Labels, actual.ObjectMeta.Labels)
-	semanticEqual = semanticEqual && equality.Semantic.DeepEqual(desired.Secrets, actual.Secrets)
 
```

Comment:
I think in this case semantic equality should mean that the desired secrets are a subset of all secrets, that way if the operator later creates a new one they get added. I think if we skip this check they never will.

Reply:
I agree, however we don't currently have a requirement for a secret. I figured we could add this logic once we did. I don't mind adding some placeholder logic though. WDYT?

Reply:
Ah, that makes sense then. It can wait until we're reconciling secrets

Reply:
I'll add a comment saying as much.